pymeasure/instruments/common_base.py

```diff
@@ -353,24 +436,24 @@ def binary_values(self, command, query_delay=0, **kwargs):
     # Property creators
     @staticmethod
     def control(  # noqa: C901 accept that this is a complex method
-        get_command,
-        set_command,
-        docs,
-        validator=lambda v, vs: v,
-        values=(),
-        map_values=False,
-        get_process=lambda v: v,
-        set_process=lambda v: v,
-        command_process=None,
-        check_set_errors=False,
-        check_get_errors=False,
-        dynamic=False,
-        preprocess_reply=None,
-        separator=',',
-        maxsplit=-1,
-        cast=float,
-        values_kwargs=None,
-        **kwargs
+            get_command,
+            set_command,
+            docs,
+            validator=lambda v, vs: v,
+            values=(),
+            map_values=False,
+            get_process=lambda v: v,
+            set_process=lambda v: v,
+            command_process=None,
+            check_set_errors=False,
+            check_get_errors=False,
+            dynamic=False,
+            preprocess_reply=None,
+            separator=',',
+            maxsplit=-1,
+            cast=float,
+            values_kwargs=None,
+            **kwargs
```

Comment:
I don't understand this indentation change? This should be indented one level from `def control`, and not two? Why is this here?

Reply:
That was an IDE auto-format setting, found it and turned it off. Fixed in last commit